pkgs/applications/science/math/geogebra/geogebra6.nix

```diff
@@ -32,7 +32,7 @@ let
         "https://download.geogebra.org/installers/6.0/GeoGebra-Linux64-Portable-${version}.zip"
         "https://web.archive.org/web/20230627211859/https://download.geogebra.org/installers/6.0/GeoGebra-Linux64-Portable-${version}.zip"
```

Comment:
I guess web archive url is broken now?

Reply:
Argh, yes, you're right, I missed that - should I take care of it?

Reply:
The reason is that `https://download.geogebra.org/installers/6.0/` keeps only 4 last releases. So some time later the primary url will be broken, that's why we need backup url to web.archive.org.

Reply:
> Hmm, interesting. I thought the reason for this being broken on stable was the removal of the older version but I suppose it should have worked?

Sure, "removal of the older version" == "keeps only 4 last releases".

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/251069